src/components/Weather/WeatherTile/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react'
+import './styles.scss'
+
+import { ThermometerIcon, UmbrellaIcon, WindIcon } from '@entur/icons'
+
+import { useWeather } from '../../../logic'
+import { WeatherIconApi } from '..'
+
+const BREAKPOINTS = {
+    fourItems: 570,
+    threeItems: 380,
+    twoItems: 290,
+}
+
+const BREAKPOINTS_COMPACT = {
+    fourItemsDesktop: 1600,
+    ThreeItemsDesktop: 1246,
+}
+
+interface Props {
+    compact?: boolean
```

Comment:
I staden for å ta inn `compact?: boolean`, kan du kanskje ta inn `breakpoints?: Breakpoints`? Då kan du bruke `BREAKPOINTS` som default, men f.eks. Compact-visningen kan overskrive med sine eigne breakpoints. Eg trur det vil skalere bedre enn om me må utvide denne komponenten dersom me vil ha værmelding i f.eks. Tidslinja på sikt.

Generelt når det gjeld design av komponentar, så bør dei ikkje ha eit forhold til kven som brukar dei eller kva kontekst dei er i. Eit lite triks eg bruker av og til er spørsmålet "Kunne dette blitt publisert på NPM/GitHub"? Då ser man ofte fort at f.eks. `compact` er eit begrep som er veldig spesifikt knytta til Tavla, men resten kunne for så vidt vore ein fin, generell WeatherTile-React-komponent.

@caroline-odden og Kristofer har forresten skrive litt om komponentdesign her: https://dev.to/selbekk/the-10-component-commandments-2a7f 😊

Reply:
Sant. Jeg ville helst ha den uten compact, men kom ikke på en god måte å gjøre det på. Compact viewet krever flere breakpoints (må gå ned igjen til tre items på et tidspunk), som gjør at det ikke er helt rett frem å bare legge det inn. Jeg skal se litt mer på det og prøve å få til noe som gjør at den ikke trenger å ta inn compact.

Reply:
Fikk gjort det du sa, men i stedet for å gi inn tall til breakpoints som `props` gir man heller inn boolske verdier, sånn at utvikleren kan velge i akkurat hvilke tilfeller man ønsker å vise de ulike ikonene. Si fra hvis det å gi inn boolske uttrykk som props ikke er bra eller imot en konvensjon!:)